### ISSUE TITLE
Certificate error due to invalid name in FluxCD HelmRelease installation

### ISSUE THREAD

**Issue by heytrav** [projectcapsule/capsule#1299]
# Bug description

When installing capsule in FluxCD as a HelmRelease all services automatically get the HelmRelease name as a prefix. This is incompatible with the certificate generated internally.

```
 Warning  ReconciliationFailed     13s (x10 over 8m14s)  kustomize-controller  Namespace/ingress-nginx dry-run failed (InternalError): Internal error occurred: failed calling webhook "owner.name
space.projectcapsule.dev": failed to call webhook: Post "https://capsule-capsule-webhook-service.capsule.svc:443/namespace-owner-reference?timeout=30s": tls: failed to verify certificate: x509: c
ertificate is valid for capsule-webhook-service.capsule.svc, not capsule-capsule-webhook-service.capsule.s
```
# How to reproduce

I'm installing capsule as a HelmRelease with the following yaml

```yaml
---
apiVersion: helm.toolkit.fluxcd.io/v2
kind: HelmRelease
metadata:
  name: capsule
  namespace: flux-system
spec:
  chart:
    spec:
      chart: capsule
      reconcileStrategy: ChartVersion
      sourceRef:
        kind: HelmRepository
        name: projectcapsule
      version: '>=0.7.2 < 0.8.0'
  install:
    crds: Create
    createNamespace: true
    remediation:
      retries: -1
  interval: 5m0s
  targetNamespace: capsule
  upgrade:
    crds: CreateReplace

```
The ingress-nginx controller which installs a little earlier attempts to make a webhook call to capsule and fails with the error mentioned earlier

# Expected behavior

I'm trying to find some way to add an additional name into SANS but so far have not been able to find something in the helm chart values.

# Logs

If applicable, please provide logs of `capsule`.

In a standard stand-alone installation of Capsule,
you'd get this by running `kubectl -n capsule-system logs deploy/capsule-controller-manager`.

# Additional context

- Helm Chart version: 0.7.2

**Comment by prometherion** [comment]
We should make this configurable, or fixed:

https://github.com/projectcapsule/capsule/blob/20807ad8f384a49da75fcb9672691d9e5dd3b38e/charts/capsule/templates/certificate.yaml#L27-L29

Are you up for a contribution? New contributors are always welcomed! 🫂 🤗

**Comment by heytrav** [comment]
Ah that's the line I was looking for.  I'll see if I can do something with this in the next few days.
Thanks!

**Comment by heytrav** [comment]
I've created https://github.com/projectcapsule/capsule/pull/1303 but after playing around with the helm chart I don't know if it's necessary.

In my case it looks like setting

```
fullnameOverride: capsule-capsule
certManager:
  generateCertificates: true
```
Solves the problem. I'll submit that PR anyway in case it might _also_ be useful to be able to set additional SANS.

**Comment by prometherion** [comment]
> I'll submit that PR anyway in case it might _also_ be useful to be able to set additional SANS

Loving it, thanks! ❤️

**Comment by prometherion** [comment]
Closed via #1303
